test/network/2020-03-01/armnetwork/models.go

```diff
@@ -10169,8 +10169,8 @@ type StringPollerResponse struct {
 // StringResponse is the response envelope for operations that return a string type.
 type StringResponse struct {
```

Comment:
I think that the `handleResponse()` will need to change for this to unmarshal into `StringResponse`, not `StringResponse.Value`.